src/Clapi/TimeDiffProtocol.hs

```diff
@@ -22,6 +22,9 @@ tdZero = TimeDelta 0.0
 getDelta :: Time -> IO TimeDelta
 getDelta theirTime = (\now -> TimeDelta $ now - (timeToFloat theirTime)) <$> getMonotonicTimeFloat
 
+tdClVal :: TimeDelta -> ClapiValue
```

Comment:
Not sure of the benefits of explicit conversions like this over using the `Clapiable` instance we already have defined? Perhaps `Clapiable` needs to be broken up direction-wise, like `ToJSON` and `FromJSON` are in `Aeson`?